homework/index.js

```diff
@@ -43,8 +90,44 @@
         });
         return;
       }
-      const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      const header = createAndAppend('header', root, {
+        class: 'header-class',
+      });
+      createAndAppend('h1', header, {
+        text: 'HYF Repository',
+        class: 'title',
+      });
+      const select = createAndAppend('select', header, {
+        class: 'select-class',
+      });
+      const mainContainer = createAndAppend('section', root, {
+        class: 'main-container',
+      });
+      const reposContainer = createAndAppend('section', mainContainer, {
+        class: 'repos-container',
+      });
+      const contributorContainer = createAndAppend('section', mainContainer, {
+        class: 'contributor-container',
+      });
+      repos
+        .sort((a, b) => {
+          return a.name.localeCompare(b.name);
+        })
+        .map((elem, index) => {
```

Comment:
You should the `.map()` method if you want to create a new array consisting of the transformed elements of an existing array. This is not the case here. Instead, you should `.forEach()` here. That will also get rid of the ESLint error that causes the Travis check to fail on this PR.

Reply:
hi @remarcmij ,I converted .map() into .forEach(). 
Thank you for the correction.